contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -1791,6 +1791,10 @@ export const Task = new IndexedDBResource({
   tableName: TABLE_NAMES.TASK,
   idField: 'task_id',
   setTasks(tasks) {
+    for (let task of tasks) {
+      // Coerce channel_id to be a simple hex string
+      task.channel_id = task.channel_id.replace('-', '');
```

Comment:
Could probably remove the `-` replacement [here](https://github.com/learningequality/studio/blob/unstable/contentcuration/contentcuration/frontend/channelEdit/vuex/task/index.js#L31) with this change